readme/apps/keyboard_interface.md

```diff
@@ -0,0 +1,48 @@
+# Keyboard interface
+
+This document includes tips for using Joplin from either:
+- a keyboard-only device (desktop)
+- a screen reader (desktop and/or mobile)
+
+## Navigating the desktop app
+
+Joplin's UI is divided into regions. In addition to the standard screen reader "jump to" shortcuts, Joplin includes its own shortcuts for navigating to these regions.
+
+See items in the "Go"/"Focus" section of the menubar for keyboard shortcuts to move focus directly to some of these regions.
+
+### Accessibility regions
+
+Joplin's main window is divided into three main regions:
+1. **Navigation 1**: The sidebar.
+2. **Navigation 2**: The notes list.
+3. **Main content**: The editor and viewer.
+
+Within the **main content**, Joplin has:
+1. **Editable**: A note title input.
+2. Two toolbars.
+3. **Editable**: A note editor.
+4. **Frame**: A note viewer.
+
+Many screen readers include shortcuts for jumping to the next/previous of each of these roles.
```

Comment:
It may make sense to link to documentation for some of the popular screen readers. For example, [Orca's structural navigation](https://help.gnome.org/users/orca/stable/howto_structural_navigation.html.en).